src/error.rs

```diff
@@ -61,21 +67,30 @@ impl From<UnalignedError> for Error {
     }
 }
 
+#[cfg(feature = "std")]
+impl From<UnalignedVecError> for Error {
+    fn from(o: UnalignedVecError) -> Error {
+        Error::UnalignedVec(o)
+    }
+}
 
-/// A slice boundary guard error, usually created by a [`Guard`](./guard/trait.Guard.html).
+/// A slice boundary guard error, usually created by a
```

Comment:
Any reason for reformatting the whole file? Why is it force-broken so close and the double newlines removed?